src/qt/txhistorydialog.cpp

```diff
@@ -352,6 +352,11 @@ int TXHistoryDialog::PopulateHistoryMap()
             displayAmount = FormatShortMP(mp_obj.getProperty(), tmpAmount) + getTokenLabel(mp_obj.getProperty());
         }
 
+        // Override - sigma spend address
+        if (type == EXODUS_TYPE_SIMPLE_SPEND) {
+            htxo.address = "Spend";
+        }
```

Comment:
Better to merge this with line 323 so it will be the only place that modifies the address.